.github/workflows/build-engine-container.yml

```
@@ -1,8 +1,5 @@
 name: build-docker
 on:
```

あ、on: push消えてdocker buildするCIが回らなくなるし、[DockerでPyinstallerビルドのエンジン使うPR](https://github.com/VOICEVOX/voicevox_engine/pull/1669/files#diff-a0a48884631c04dba63cc4a887b4c526c43b3b6a69e134cffef113dfc6ee6233R3-R5)の変更こっちに置き換えてもいいかも？
```suggestion
on:
  # FIXME: masterブランチへのpush時にビルド工程をテストする
  # https://github.com/VOICEVOX/voicevox_engine/pull/1669#discussion_r2076730954
  workflow_call:
```
こうでも良いかも
```suggestion
on:
  # FIXME: デフォルトブランチへのpush時にビルドをテストする
  # https://github.com/VOICEVOX/voicevox_engine/pull/1669#discussion_r2076730954
  workflow_call:
```